src/vivarium_census_prl_synth_pop/components/observers.py

```diff
@@ -752,6 +782,7 @@ def get_observation(self, event: Event) -> pd.DataFrame:
             pop[pop["has_ssn"]].groupby("address_id").apply(lambda df_g: list(df_g.index))
         )
         household_members_w_ssn = simulants_wo_ssn.map(household_members_w_ssn).dropna()
+        # todo: Fix me MIC-4092
```

Comment:
I talked to Abie and am not going to make a change to this but essentially we initialize a np rng generator here and use vivarium random seed so the np generator is the same on each time step because the seed doesn't change. For comparison in the utils function I wrote I have the seed I create from _key. He also confirmed to leave both approaches and hopes RT will determine at some point which approach is better but for now both implementations are acceptable. I will remove the TODO.